src/components/menu-list/menu-list.scss

```diff
@@ -20,4 +20,10 @@
             margin-right: functions.pxToRem(14);
         }
     }
+
+    hr.mdc-deprecated-list-divider {
```

Comment:
`menu-list` is an internal component, so a reader of the changelog probably has no idea what it is. The scope for this commit should just be `menu`, as that's the public component that's affected 🙂

Reply:
Nice. Gotcha!